tests/test_agent.py

```diff
@@ -66,6 +66,7 @@ def test_erroring(mock_send_report, handler_that_errors, mock_context):
 
     assert iopipe.report.report['errors']['name'] == 'ValueError'
     assert iopipe.report.report['errors']['message'] == 'Behold, a value error'
+    assert isinstance(iopipe.report.report['errors']['stack'], str)
```

Comment:
Question: Did this test fail before you added your code that fixes it?

Reply:
No, it did not fail before. It is just an assertion that should be true.

Reply:
@pselle Did that answer your question?